inc/stat.class.php

```diff
@@ -1909,11 +1912,10 @@ private function generateCsvFile($labels, $series, $options = []) {
          ksort($values);
 
          if (!count($headers) && $options['title']) {
-            $headers[] = $options['title'];
+            $headers = $options['title'];
          }
 
          // Print labels
-         fwrite($fp, $_SESSION["glpicsv_delimiter"]);
```

Comment:
I guess this was made to have an empty value for the first column header. This should not be changed.

Reply:
the issue is that with if we have 'name' => __('Value') then the data should be json object and not number array, therefore there is quite some code change